### PR TITLE
Add portmgr entry for 2022q2

### PR DESCRIPTION
**Pull request by rene0**

### CODE REVIEW

File: 2022q2/portmgr.adoc
Comment (pauamma) [inline 2022q2/portmgr.adoc:30]: "incompatibilities kernel module ports" Missing "in" here, or perhaps "between".

File: 2022q2/portmgr.adoc
Comment (pauamma) [inline 2022q2/portmgr.adoc:33]: "in the Ports Tree"

Also, either "fix any documentation-related error" or "make any documentation-related change"

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2022q2/portmgr.adoc
Comment (rene0) [reply 2022q2/portmgr.adoc]: Fixed

File: 2022q2/portmgr.adoc
Comment (rene0) [reply 2022q2/portmgr.adoc]: Also fixed, thanks for the review.

**Review by pauamma — APPROVED** [review]
Comment can be removed on or after merging.

**Comment by rene0** [comment]
Feel free to commit, I don't have write access here iirc.

**Pull request merged**
